Source/Core/Core/IOS/FS/FileSystemProxy.cpp

```diff
@@ -108,7 +109,7 @@ IPCCommandResult FS::Open(const OpenRequest& request)
 
   auto backend_fd = m_ios.GetFS()->OpenFile(request.uid, request.gid, request.path,
                                             static_cast<Mode>(request.flags & 3));
-  LogResult(StringFromFormat("OpenFile(%s)", request.path.c_str()), backend_fd);
+  LogResult(fmt::format("OpenFile({})", request.path), backend_fd);
```

Comment:
That kinda sounds like a candidate for a `printf`-like signature...does `fmt` have something like that?
Like `LogResult(result, "OpenFile({})", request.path)`

Reply:
Yeah, format can be used like that via parameter packs.